### ISSUE TITLE
Users are being imported from LDAP every time getGroupMembers is called 

### ISSUE THREAD

**Issue by JorgenPo** [keycloak/keycloak#33147]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ldap

### Describe the bug

We have ~500 members in a LDAP group and we do periodic group member querying in our code. As a result, we are observing a lot of user attribute update queries to our DB.
It seems like Keycloak imports all users from LDAP when requesting group members. Users are not checked in DB for existence. What's more, keycloak makes DB UPDATE requests when updating LDAP_ID and LDAP_ENTRY_DN attributes after every import. 
We set up keycloak to periodically sync with ldap and don't want Keycloak to import all group users every time we request for group members.
Does Keycloak actually need to import all users every time we request group members? Could you please consider making it optional if so?

My question/problem may be related to the issue https://github.com/keycloak/keycloak/issues/32870. Can we apply the similar fix to handle that particular case too?

Nevertheless, thank you for the great piece of software!

### Version

Tested on the latest main

### Regression

- [ ] The issue is a regression

### Expected behavior

When requesting group members keycloak only imports users that are not found in the DB 

### Actual behavior

All group member users are imported from LDAP and their attributes are updated in the DB

### How to Reproduce?

Set up group mapper from LDAP and request for group members. 

### Anything else?

Please ask for additional logs/details if needed

**Comment by sguilhen** [comment]
@JorgenPo I think it is exactly the same as https://github.com/keycloak/keycloak/issues/31786. We have a PR currently being reviewed for that, so I would expect much better results once that one is merged.

**Comment by JorgenPo** [comment]
@sguilhen, seems like it is indeed. Glad it fixed now. Thank you for pointing this out.

**Comment by jonkoops** [comment]
Duplicate of #31786
